dotcom-rendering/src/components/Picture.tsx

```diff
@@ -79,6 +79,19 @@ const decideImageWidths = ({
 				];
 		}
 	}
+	if (format.design === ArticleDesign.Picture) {
+		// the order is important here. Picture content type images come through as main media, so needs to appear
+		// above `isMainMedia`, so the images do not appear low quality.
```

Comment:
If this is true, why not place it inside of the `if (isMainMedia)` block?

Reply:
It seemed safer to create a separate case. I'm not sure if, in future, Picture articles might serve images in the body rather than main block